### PR TITLE
Issues/bug record payement invoice

### PR DESCRIPTION
**Pull request by polymahh**
## Fix:
added new path for paying invoices `invoice/pay`
fixed the record payment page and state 
fixed payment invoice success message "backend"

https://www.loom.com/share/21e1d6eca50a4274a80916da105420be?sid=f8d9bcb8-e20c-4396-91e1-8a66bb828a71

### CODE REVIEW

**Comment by salahlalami** [comment]
Thank you , @polymahh 

@onfranciis his fixing invoice payment path [https://github.com/idurar/idurar-erp-crm/pull/410](https://github.com/idurar/idurar-erp-crm/pull/410) , please wait until he merge hi pr

**Review by onfranciis — APPROVED** [review]

**Review by salahlalami — APPROVED** [review]

**Pull request merged**
